x-pack/plugins/asset_inventory/public/components/app.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { FormattedMessage, I18nProvider } from '@kbn/i18n-react';
+import { BrowserRouter as Router } from '@kbn/shared-ux-router';
+import { EuiPageTemplate, EuiTitle } from '@elastic/eui';
+import type { CoreStart } from '@kbn/core/public';
+import type { NavigationPublicPluginStart } from '@kbn/navigation-plugin/public';
+
+import { PLUGIN_ID } from '../../common';
+
+interface AssetInventoryAppDeps {
+  basename: string;
+  notifications: CoreStart['notifications'];
+  http: CoreStart['http'];
+  navigation: NavigationPublicPluginStart;
+}
+
+export const AssetInventoryApp = ({ basename, navigation }: AssetInventoryAppDeps) => {
+  // Note that `navigation.ui.TopNavMenu` is a stateful component exported on the `navigation` plugin's start contract.
+  return (
+    <Router basename={basename}>
+      <I18nProvider>
+        <>
+          <navigation.ui.TopNavMenu
+            appName={PLUGIN_ID}
+            showSearchBar={true}
+            useDefaultBehaviors={true}
+          />
+          <EuiPageTemplate restrictWidth="1000px">
```

Comment:
Why are we restricting the width to `1000px`?

🤔 Should we use break points here?
https://eui.elastic.co/#/theming/breakpoints/values

Reply:
This was auto-generated by the script. I agree we should change it but the UI is not relevant at this point. We just wanted to have an empty plugin to start working upon it